tee-worker/litentry/core/data-providers/src/achainable.rs

```diff
@@ -320,27 +142,26 @@ impl AchainablePost for AchainableClient {
 			self.client.post_capture::<ReqParams, ReqBody, serde_json::Value>(params, body);
 		debug!("ReqBody response: {:?}", response);
 		match response {
-			Ok(res) =>
-				if let Some(value) = res.get("result") {
-					Ok(value.clone())
-				} else {
-					Err(Error::AchainableError("Invalid response".to_string()))
-				},
+			Ok(res) => Ok(res),
 			Err(e) => Err(Error::RequestError(format!("{:?}", e))),
 		}
 	}
 }
 
-pub trait AchainableResultParser {
+pub trait AchainableSystemLabelParser {
 	type Item;
 	fn parse(value: serde_json::Value) -> Result<Self::Item, Error>;
 }
 
-impl AchainableResultParser for AchainableClient {
+impl AchainableSystemLabelParser for AchainableClient {
 	type Item = bool;
-	fn parse(value: serde_json::Value) -> Result<Self::Item, Error> {
-		if let Some(b) = value.as_bool() {
-			Ok(b)
+	fn parse(response: serde_json::Value) -> Result<Self::Item, Error> {
+		if let Some(value) = response.get("result") {
+			if let Some(b) = value.as_bool() {
+				Ok(b)
+			} else {
+				Err(Error::AchainableError("Invalid response".to_string()))
```

Comment:
Can we give more detail error message? At least to differentiate from the follow one.